client/components/migration/feature-announcement.jsx

```diff
@@ -62,7 +62,7 @@ const FeatureAnnouncement = ( { translate, isEligable, previousMigrationState }
 						{translate('A new dedicated WooCommerce Shipping extension is now available')}
 					</h2>
 					<p>{translate(
-						'WooCommerce Shipping and WooCommerce Tax are now two dedicated extensions. We\'ll automatically deactivate WooCommerce Shipping & Tax and carry over your settings when you update.')}</p>
+						'WooCommerce Shipping is now a new dedicated extension. We\'ll allow you to carry over your settings and shipping labels when you update.')}</p>
```

Comment:
wouldn't `We’ll ensure a smooth transition by automatically carrying over all your settings and shipping labels when you update.`  be a better replacement for `We'll allow ...` ? or if we don't want to do it automatically, we can probably say the following:  `We’ll ensure a smooth transition by providing the functionality to carry over all your settings and shipping labels when you update.`.
I'm mostly not very happy with seeing `allow` in the sentence suggested by this PR, as it's their data after all.

Reply:
Just went with your latest suggestion. As you said, I wanted to convey that the migration is not automatic, and you put it perfectly in that phrase. Thanks!